dotcom-rendering/src/web/browser/bork.ts

```diff
@@ -0,0 +1,16 @@
+/** synthetically bork FID https://web.dev/fid/ */
```

Comment:
why this file and the inline script?

Reply:
I need to pick one…

Inline:
- synchronous and before anything else
- no risk of error propagation 

File:
- typescript checks
- errors need to be handled gracefully
- loads with everything else

---

Maybe the best of both worlds is to create a separate file that exports a function on which to call `toString()`?